src/components/Toolbar.vue

```diff
@@ -0,0 +1,76 @@
+<template lang="pug" xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
+ div
+  v-toolbar(clipped-left fixed app)
+    v-toolbar-side-icon(@click.stop="drawer= !drawer")
+    v-toolbar-title {{ title }}
+    v-spacer
+    v-toolbar-items(hidden-sm-and-down)
+      v-btn(flat)
+        v-icon mdi-forum
+        span Forum
+      v-btn(flat href="../konnekt")
+        v-icon mdi-web
```

Comment:
Change `v-icon mdi-web` to `v-icon(left) mdi-web`

Reply:
done